rock-paper-scissor-game/index.js

```diff
@@ -0,0 +1,33 @@
+var comp_score = 0;
+var user_score = 0;
+
+
+function check(str) {
+    var vals = ['rock', 'paper', 'scissor'];
+    var index = Math.floor((Math.random() * 3));
+    var comp_res = vals[index];
+
+    if (str == comp_res) {
+        document.getElementById('result').innerHTML="Draw!";
+    }
+    else if(str == 'rock' && comp_res == 'scissor') {
+        document.getElementById('result').innerHTML='You Won';
+        user_score += 1;
+        document.getElementById('user_score_display').innerHTML=user_score;
+    }
+    else if(str == 'paper' && comp_res == 'rock') {
+        document.getElementById('result').innerHTML='You Won';
+        user_score += 1;
+        document.getElementById('user_score_display').innerHTML=user_score;
+    }
+    else if(str == 'scissor' && comp_res == 'paper') {
+        document.getElementById('result').innerHTML="You Won";
+        user_score += 1;
+        document.getElementById('user_score_display').innerHTML=user_score;
+    }
+    else {
+        document.getElementById('result').innerHTML='You Lose';
+        comp_score += 1;
+        document.getElementById('comp_score_display').innerHTML=comp_score;
+    }
```

Comment:
instead of multiple if..else statemets and repeating code section use concept of ternary operator:
what i can see is there are only three cases when user can win:
you_win = ( (you === "rock") && (opponent === "scissor") ||
(you === "paper") && (opponent === "rock") ||
(you === "scissor") && (opponent === "rock") ) // returns true for your win cases only.
.
.
so (you === opponent) ? "draw message" : you_win ? call a method for updating score and message for user : call method for updating score and message fro computer;